Tests/TestingTests/MiscellaneousTests.swift

```diff
@@ -405,41 +409,49 @@ struct MiscellaneousTests {
 
   @Test("Properties related to parameterization")
   func parameterizationRelatedProperties() async throws {
-    let typeTest = Test.__type(SendableTests.self, displayName: "", traits: [], sourceLocation: .init())
-    #expect(!typeTest.isParameterized)
-    #expect(typeTest.testCases == nil)
-    #expect(typeTest.parameters == nil)
-
-    let monomorphicTestFunction = Test {}
-    #expect(!monomorphicTestFunction.isParameterized)
-    let monomorphicTestFunctionTestCases = try #require(monomorphicTestFunction.testCases)
-    #expect(monomorphicTestFunctionTestCases.underestimatedCount == 1)
-    let monomorphicTestFunctionParameters = try #require(monomorphicTestFunction.parameters)
-    #expect(monomorphicTestFunctionParameters.isEmpty)
-
-    let parameterizedTestFunction = Test(arguments: 0 ..< 100, parameters: [Test.Parameter(index: 0, firstName: "i", type: Int.self)]) { _ in }
-    #expect(parameterizedTestFunction.isParameterized)
-    let parameterizedTestFunctionTestCases = try #require(parameterizedTestFunction.testCases)
-    #expect(parameterizedTestFunctionTestCases.underestimatedCount == 100)
-    let parameterizedTestFunctionParameters = try #require(parameterizedTestFunction.parameters)
-    #expect(parameterizedTestFunctionParameters.count == 1)
-    let parameterizedTestFunctionFirstParameter = try #require(parameterizedTestFunctionParameters.first)
-    #expect(parameterizedTestFunctionFirstParameter.firstName == "i")
-
-    let parameterizedTestFunction2 = Test(arguments: 0 ..< 100, 0 ..< 100, parameters: [
-      Test.Parameter(index: 0, firstName: "i", type: Int.self),
-      Test.Parameter(index: 1, firstName: "j", secondName: "value", type: Int.self),
-    ]) { _, _ in }
-    #expect(parameterizedTestFunction2.isParameterized)
-    let parameterizedTestFunction2TestCases = try #require(parameterizedTestFunction2.testCases)
-    #expect(parameterizedTestFunction2TestCases.underestimatedCount == 100 * 100)
-    let parameterizedTestFunction2Parameters = try #require(parameterizedTestFunction2.parameters)
-    #expect(parameterizedTestFunction2Parameters.count == 2)
-    let parameterizedTestFunction2FirstParameter = try #require(parameterizedTestFunction2Parameters.first)
-    #expect(parameterizedTestFunction2FirstParameter.firstName == "i")
-    let parameterizedTestFunction2SecondParameter = try #require(parameterizedTestFunction2Parameters.last)
-    #expect(parameterizedTestFunction2SecondParameter.firstName == "j")
-    #expect(parameterizedTestFunction2SecondParameter.secondName == "value")
+    do {
```

Comment:
Please double-check that we don't regress code coverage here. (I'm not saying we do, just being cautious.)